webapp/src/features/intersections/reports/report-list-table.tsx

```diff
@@ -1,56 +1,18 @@
 import React from 'react'
 import { format } from 'date-fns'
-import {
-  Avatar,
-  Box,
-  Button,
-  IconButton,
-  Table,
-  TableBody,
-  TableCell,
-  TablePagination,
-  TableRow,
-  Typography,
-} from '@mui/material'
+import { Box, Button, Table, TableBody, TableCell, TablePagination, TableRow, Typography } from '@mui/material'
```

Comment:
For these horizontal vs vertical include list, is that a setting we can set to keep consistent?

Reply:
Right now, it condenses if it all fits onto one line (length 120), and expands to new lines if it doesn't. It looks like we can also add eslint, which will expand every import onto multiple lines - is that a better solution?